pkg/cli/cliflags/flags.go

```diff
@@ -228,12 +228,19 @@ server to listen on an external address in insecure mode.`,
 		Description: `Define the maximum number of results that will be retrieved.`,
 	}
 
+	SetPassword = FlagInfo{
+		Name:   "set-password",
+		EnvVar: "COCKROACH_SET_PASSWORD",
+		Description: `
+The created user's password. If provided, disables prompting. Pass '-' to
```

Comment:
This comment still mentions passing `-` for standard input, which isn't implemented (and shouldn't be: we shouldn't interpret anything in the user's password. If we want to support reading from stdin it should be a separate flag like `--set-password-from-file=filename`, which could also take `-`)